packages/nimble-components/src/wafer-map/styles.ts

```diff
@@ -58,4 +58,30 @@ export const styles = css`
         display: inline-block;
         position: absolute;
     }
+
+    .hover-layer {
+        position: absolute;
+        pointer-events: none;
+        width: 100%;
+        height: 100%;
+    }
+
+    .hover-rect {
+        fill: transparent;
+        stroke: white;
+        outline-style: solid;
+        outline-color: green;
```

Comment:
Can you use the `import { DigitalGreenLight} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';` token instead:
```suggestion
        outline-color: ${DigitalGreenLight};
```

That aligns with the borderHoverColor token:
https://github.com/ni/nimble/blob/d25f8eea7d2530431f1d1a31c92d29aee22f7644/packages/nimble-components/src/theme-provider/design-tokens.ts#L169-L171